config/locales/en.yml

```diff
@@ -43,6 +43,7 @@ en:
                 other: 'Missing required oauth fields in manifest: %{missing_keys}'
             missing_source: Could not find app.js
             style_in_template: "<style> tag in %{template}. Use an app.css file instead."
+            no_code_for_ifo_notemplate: Javascripts, stylesheets and templates are not allowed when an iframe URI or noTemplate is specified
```

Comment:
`noTemplate` is a special key that people can declare to be true of false. It should be written together and not localised.